lib/twap/util/generate_order.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 
+const _isFinite = require('lodash/isFinite')
```

Comment:
can you add a unit test for `generate_order.js`? should also cover the float issue